content/vi/docs/tasks/tools/install-minikube.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Cài đặt Minikube
+content_template: templates/task
+weight: 20
+card:
+  name: tasks
+  weight: 10
+---
+
+{{% capture overview %}}
+
+Tài liệu này sẽ chỉ cho bạn các cài đặt [Minikube](/docs/tutorials/hello-minikube), một công cụ chạy một single-node Kubernetes cluster trong một máy ảo (VM) trên máy tính của bạn.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+{{< tabs name="minikube_before_you_begin" >}}
+{{% tab name="Linux" %}}
+Để kiểm tra hỗ trợ ảo hóa trên Linux, hãy chạy câu lệnh sau và xác nhận có giá trị kết quả đầu ra.
+```
+grep -E --color 'vmx|svm' /proc/cpuinfo
+```
+{{% /tab %}}
+
+{{% tab name="macOS" %}}
+Để kiểm tra hỗ trợ ảo hóa trên macOS, hãy chạy câu lệnh sau trên cửa sổ terminal của bạn.
+```
+sysctl -a | grep -E --color 'machdep.cpu.features|VMX' 
+```
+Nếu bạn nhìn thấy `VMX` ở kết quả (có màu), thì VT-x đã được enable trên máy của bạn.
+{{% /tab %}}
+
+{{% tab name="Windows" %}}
+Để kiểm tra hỗ trợ ảo hóa trên Windows 8 hay phiên bản Windows cao hơn, hãy chạy câu lệnh sau trên terminal của Window hoặc command prompt.
+```
+systeminfo
+```
+Nếu bạn nhìn thấy những thông tin đầu ra sau, tức là ảo hóa được hỗ trợ trên máy tính của bạn.
+```
+Hyper-V Requirements:     VM Monitor Mode Extensions: Yes
+                          Virtualization Enabled In Firmware: Yes
+                          Second Level Address Translation: Yes
+                          Data Execution Prevention Available: Yes
+```
+
+Nếu bạn nhìn thấy những thông tin đầu ra sau, tức là máy tính của bạn đã được cài đặt Hypervisor và bạn có thể bỏ qua những bước tiếp theo.
```

Comment:
Should be:
"Nếu bạn thấy thông tin sau, thì hệ thống đã được cài đặt Hypervisor và bạn có thể bỏ qua bước tiếp theo."